pkg/chains/formats/slsa/README.md

```diff
@@ -12,4 +12,314 @@ Shown below is the mapping between Tekton chains proveance and SLSA predicate.
 |Tekton Chains Provenance Format version     | SLSA predicate | Notes |
 |:------------------------------------------|---------------:|------:|
 |**slsa/v1**| **slsa v0.2**  | same as currently supported `in-toto` format|
-|**slsa/v2alpha1**| **slsa v0.2**  | contains complete build instructions as in [TEP0122](https://github.com/tektoncd/community/pull/820). This is still a WIP and currently only available for taskrun level provenance. |
+|**slsa/v2alpha1**| **slsa v0.2**  | contains complete build instructions as in [TEP0122](https://github.com/tektoncd/community/pull/820) and the [example below](#provenance-format-example-for-slsa/v2alpha1). This is still a WIP and currently only available for taskrun level provenance. |
+
+## Provenance Format Example for slsa/v2alpha1
+When running the following taskrun with bundle resolver:
+
+```yaml
+apiVersion: tekton.dev/v1beta1
+kind: TaskRun
+metadata:
+  generateName: bundles-resolver-
+spec:
+  workspaces:
+    - name: output
+      emptyDir: {}
+  podTemplate:
+    securityContext:
+      fsGroup: 65532
+  taskRef:
+    resolver: bundles
+    params:
+      - name: bundle
+        value: gcr.io/tekton-releases/catalog/upstream/git-clone@sha256:8e2c3fb0f719d6463e950f3e44965aa314e69b800833e29e68ba2616bb82deeb
+      - name: name
+        value: git-clone
+      - name: kind
+        value: task
+  params:
+    - name: url
+      value: https://github.com/kelseyhightower/nocode
+    - name: revision
+      value: master
+```
+
```

Comment:
Can we add a [link](https://github.com/tektoncd/catalog/tree/main/task/git-clone) to the referenced git-clone task yaml aside the taskrun def so that readers can get a sense of what the referenced task looks like with a click?

Reply:
Can do!